build/README.md

```diff
@@ -93,14 +93,14 @@ Run `go get -u <dependency>`. To get a specific version, run `go get -u <depende
 When updating a dependency, you should run `go mod tidy` after `go get` to ensure the old entries
 are removed from go.sum.
 
-You must then run `make -k vendor_rebuild` to ensure the modules are installed. These changes must
```

Comment:
you still need the `-k` if your vendor gets into a weird state and execgen won't run.

Reply:
Really? `make vendor_rebuild` shouldn't be building `execgen`.

Reply:
I just tested this:

```
~ rm bin/execgen*
~ make vendor_rebuild
...
~ ls bin/execgen*
ls: bin/execgen*: No such file or directory
```

Reply:
weird, it does that on my system.

```
make vendor_rebuild
Running make with -j8
GOPATH set to /Users/otan/go
go install -v execgen
bin/prereqs ./pkg/sql/colexec/execgen/cmd/execgen > bin/execgen.d.tmp
```

Reply:
Are you running on top of https://github.com/cockroachdb/cockroach/pull/50636?

Reply:
ah, nope!